.circleci/config.yml

```diff
@@ -31,7 +31,7 @@ e2e-defaults: &e2e-defaults
   docker:
     - image: celohq/circleci:geth1.9-2
   environment:
-    CELO_BLOCKCHAIN_BRANCH_TO_TEST: master
```

Comment:
This probably should be back to master right?

Reply:
Not for the moment, since it's not merged there yet

Reply:
I guess for this to merge into master, this needs to be master no?